webpack.config.js

```diff
@@ -60,7 +60,8 @@ let config = dev.getDefaultWebpackConfig({
         new CopyWebpackPlugin([
             // getCoreNodeModule.js -> dist/node_modules/getCoreNodeModule.js
             { from: './out/src/utils/getCoreNodeModule.js', to: 'node_modules' },
-        ]),
```

Comment:
Haha how did this change happen?

Reply:
I was testing some in this file and missed this change. Thanks for catching.